app/http.php

```diff
@@ -58,73 +58,49 @@
     $app = new App('UTC');
 
     go(function () use ($register, $app) {
-        // wait for database to be ready
-        $attempts = 0;
-        $max = 10;
-        $sleep = 1;
 
-        do {
-            try {
-                $attempts++;
-                $db = $register->get('dbPool')->get();
-                $redis = $register->get('redisPool')->get();
-                break; // leave the do-while if successful
-            } catch (\Exception $e) {
-                Console::warning("Database not ready. Retrying connection ({$attempts})...");
-                if ($attempts >= $max) {
-                    throw new \Exception('Failed to connect to database: ' . $e->getMessage());
-                }
-                sleep($sleep);
-            }
-        } while ($attempts < $max);
-
-        App::setResource('db', fn() => $db);
+        $redis = $register->get('redisPool')->get();
         App::setResource('cache', fn() => $redis);
 
-        $dbForConsole = $app->getResource('dbForConsole'); /** @var Utopia\Database\Database $dbForConsole */
+        $dbPool = $register->get('dbPool');
+        [$database, $returnDatabase] = $dbPool->getDBFromPool('console', $redis);
```

Comment:
I don't like this syntax, I think we can still use the registry to get individual connections, just using the pool instead of how it was.